lib/ansible/modules/windows/win_psmodule.ps1

```diff
@@ -9,18 +10,33 @@
 
 $params = Parse-Args $args -supports_check_mode $true
 
-$name = Get-AnsibleParam -obj $params -name "name" -type "str" -failifempty $true
+$name = Get-AnsibleParam -obj $params -name "name" -type "str" -failifempty $true 
 $repo = Get-AnsibleParam -obj $params -name "repository" -type "str"
 $url = Get-AnsibleParam -obj $params -name "url" -type "str"
 $state = Get-AnsibleParam -obj $params -name "state" -type "str" -default "present" -validateset "present", "absent"
 $allow_clobber = Get-AnsibleParam -obj $params -name "allow_clobber" -type "bool" -default $false
 $check_mode = Get-AnsibleParam -obj $params -name "_ansible_check_mode" -default $false
+$latest = Get-AnsibleParam -obj $params -name "latest" -type "bool" -default $false
```

Comment:
Wondering whether we should have `latest` as a state like we do with a few other modules like `win_chocolatey`

Reply:
yes, sounds better, will implement